release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -705,6 +707,8 @@ The MetalLB `AddressPool` custom resource definition (CRD) has been deprecated f
 [id="ocp-4-16-scalability-and-performance-bug-fixes_{context}"]
 ==== Scalability and performance
 
+* Previously, the NUMA Resources Operator did not correctly handle `KubeletConfig` objects featuring an empty payload. As a consequence, the NUMA resources controller reported a `CrashLoopBackOff` status when applying a `NUMAResourcesOperator` resource in this scenario. With this release, improved filtering logic ensures that the NUMA Resources Operator handles `KubeletConfig` objects with empty payloads as expected. (link:https://issues.redhat.com/browse/OCPBUGS-30236[*OCPBUGS-30236*])
+
```

Comment:
@ffromani - can you review this bug fix text please?

Reply:
LGTM